iotile_ext_cloud/test/test_login.py

```diff
@@ -86,3 +86,24 @@ def test_alternative_domains(registry):
         cloud.refresh_token()
 
     assert registry.get_config('arch:cloud_token') == 'new-token'
+
+def test_check_time():
```

Comment:
This test is good but we should add one that should evaluate to true as well.

Since we have 300 seconds of slop, you could just capture utcnow() from the datetime module and create the header dynamically in the test counting on the test taking < 300 seconds to run.  

You can use `strftime` to create the right format for the header:
https://docs.python.org/2/library/datetime.html#strftime-and-strptime-behavior